### ISSUE TITLE
Create report without setting force to true?

### ISSUE THREAD

**Issue by galvarez421** [yargalot/grunt-accessibility#42]
Is there a way to have the task create a report without setting the `force` option to true?

I am trying to achieve a grunt `build` task that will fail if the accessibility task fails due to accessibility notices, warnings, or errors being discovered, but being able to view the report in this case would be very beneficial. It seems that the only way to have the task create a report after notices, warnings, or errors are discovered is to set `force` to true, right?

**Comment by prantlf** [comment]
Fixed by [a similar issue in AccessSniff](https://github.com/yargalot/AccessSniff/issues/41), PRs [42](https://github.com/yargalot/AccessSniff/pull/42) and [43](https://github.com/yargalot/AccessSniff/pull/43).

Before this makes it to upstream, I published an alternative module including this feature and upgraded dependencies - [grunt-accessibility-ext](https://github.com/yargalot/grunt-accessibility). Unfortunately, this module needs a compilation step before publishing and it is not possible to depend on a forked Github branch.

**Comment by yargalot** [comment]
Fixed with release 6
